libs/platform/src/lib/components/form/select/commons/base-select.ts

```diff
@@ -177,12 +176,17 @@ export abstract class BaseSelect extends CollectionBaseInput implements AfterVie
     maxWidth?: number;
 
     /**
-     * content Density of element. 'cozy' | 'compact'
+     * content Density of element. 'cozy' | 'compact'| 'condensed'
```

Comment:
I don't think this component has `condensed` content density. It should be only cozy and compact.